libs/platform/auth/src/services/anon-auth-token.service.ts

```diff
@@ -41,7 +43,11 @@ export class AnonAuthTokenService implements AuthTokenService {
       .get<string>(this.ANONYMOUS_USER_IDENTIFIER, StorageType.Session)
       .pipe(
         switchMap((userId) => {
-          return userId ? of(userId) : this.createAnonymousId();
+          return featureVersion >= '1.4'
+            ? of(userId)
+            : userId
+            ? of(userId)
+            : this.createAnonymousId();
```

Comment:
```suggestion
          return featureVersion >= '1.4' || userId
            ? of(userId)
            : this.createAnonymousId();
```

Reply:
but seems that we do not need this, as we're no longer using this service in >= 1.4

Reply:
👍